frontend/src/modules/displays/index.ts

```diff
@@ -1,3 +1,9 @@
+/**
+ * @summary A short one-line description for the file
```

Comment:
These placeholders need to be filled in. See also my comments on `modules/app/index.ts`. What is a "display"? What kinds of files should be in this folder, vs. in other folders?

Reply:
I am updating the following
```
/**
 * @summary UI resuable components used in app
 *
 * @file Components which can be reused across app
 * 
 * Currently it has Carousel(used in Landing page to show slides of content)
 * ModeBanner(Current mode text shown in footer of quickstart./set alarms page)
 * ToogleValue(Used in Settings page to toggle values for theme color & unit) 
 * ValueDisplay(All the numeric values shown across app uses this component )
 * ValueSelectorDisplay (Its similar to `ValueDisplay` except that it accepts 
 * selector as input & shows the value fetching from redux store using selector )
 */
```